build/secondary/third_party/libwebp/BUILD.gn

```diff
@@ -0,0 +1,121 @@
+# Copyright 2017 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+config("libwebp_config") {
+  include_dirs = [ "src" ]
```

Comment:
You added this as an include directory but none of the files listed below in the source set are headers. Is this necessary? Or maybe you missed adding headers to the sources below.